### ISSUE TITLE
Multiselect @input called twice 

### ISSUE THREAD

**Issue by raimund-schluessler** [nextcloud/nextcloud-vue#784]
A multiselect like this

```html
<multiselect
	id="users-groups-search"
	:options="usersOrGroups"
	:searchable="true"
	:internal-search="false"
	:max-height="600"
	:show-no-results="true"
	:placeholder="placeholder"
	:class="{ 'showContent': inputGiven, 'icon-loading': isLoading }"
	:user-select="true"
	open-direction="bottom"
	track-by="user"
	label="user"
	@search-change="findSharee"
	@input="shareCalendar"
/>
```

triggers the `@input` method twice when an option is selected. Originally reported in https://github.com/nextcloud/tasks/issues/778.

Edit: This also happens for multiselects with other options, not only user-select.

**Comment by raimund-schluessler** [comment]
Just guessing, but I think it has something to do with https://github.com/nextcloud/nextcloud-vue/pull/601

**Comment by raimund-schluessler** [comment]
Could it be that this https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Multiselect/Multiselect.vue#L318 is not necessary?

**Comment by skjnldsv** [comment]
Maybe it is just set twice?

**Comment by raimund-schluessler** [comment]
It only happens since I updated vue components. See also https://github.com/nextcloud/contacts/pull/1389 where it now happens as well.

**Comment by skjnldsv** [comment]
Maybe it is an update from vue-multiselect?
Can you trace down the event emitions?

**Comment by raimund-schluessler** [comment]
`console.trace()` in the `@input`-callback gives this for the first call:

```log
console.trace() CalendarShare.vue:107
    shareCalendar CalendarShare.vue:107
    VueJS 4
        invokeWithErrorHandling
        invoker
        invokeWithErrorHandling
        $emit
    set Multiselect.js:109
    callback Multiselect.js:109
    VueJS 4
        invokeWithErrorHandling
        invoker
        invokeWithErrorHandling
        $emit
    select vue-multiselect.min.js:1
    click vue-multiselect.min.js:1
    VueJS 3
        invokeWithErrorHandling
        invoker
        _wrapper
```

And this for the second call:
```log
console.trace() CalendarShare.vue:107
    shareCalendar CalendarShare.vue:107
    VueJS 6
        invokeWithErrorHandling
        invoker
        invokeWithErrorHandling
        invoker
        invokeWithErrorHandling
        $emit
    select vue-multiselect.min.js:1
    click vue-multiselect.min.js:1
    VueJS 3
        invokeWithErrorHandling
        invoker
        _wrapper
```

This happen with Tasks ce6f2afa152ae60cc26d07be89d52eb5b6e72b08, I just added `console.trace` here https://github.com/nextcloud/tasks/blob/master/src/components/CalendarShare.vue#L102

**Comment by raimund-schluessler** [comment]
Add a test using https://jestjs.io/docs/en/expect#tohavebeencalledtimesnumber

**Comment by raimund-schluessler** [comment]
Test to reproduce the issue is in #844.

**Comment by raimund-schluessler** [comment]
Closed with #844.
